dev/io.openliberty.microprofile.opentracing.2.0.internal_fat_tck/fat/src/io/openliberty/microprofile/opentracing/internal/tck/OpentracingTCKLauncher.java

```diff
@@ -30,30 +30,28 @@
 @RunWith(FATRunner.class)
 public class OpentracingTCKLauncher {
 
-    @Server("OpentracingTCKServer")
-    public static LibertyServer server;
+	@Server("OpentracingTCKServer")
+	public static LibertyServer server;
 
-    @BeforeClass
-    public static void setUp() throws Exception {
-        server.startServer();
-    }
+	@BeforeClass
+	public static void setUp() throws Exception {
+		server.startServer();
+	}
 
-    /*
-     * CWWKG0014E - Ignore due to server.xml intermittently missing
-     */
-    @AfterClass
-    public static void tearDown() throws Exception {
-        server.stopServer("CWMOT0009W", "CWWKG0014E", "CWPMI2005W");
-    }
+	/*
+	 * CWWKG0014E - Ignore due to server.xml intermittently missing
+	 */
+	@AfterClass
+	public static void tearDown() throws Exception {
+		server.stopServer("CWMOT0009W", "CWWKG0014E", "CWPMI2005W");
+	}
 
-    @Test
-    @AllowedFFDC // The tested deployment exceptions cause FFDC so we have to allow for this.
-    public void launchOpenTracing20Tck() throws Exception {
-        // Use default tck-suite.xml
-        String bucketName = "io.openliberty.opentracing.2.0.internal_fat_tck";
-        String testName = this.getClass() + ":launchOpenTracing20Tck";
-        Type type = Type.MICROPROFILE;
-        String specName = "Open Tracing";
-        TCKRunner.runTCK(server, bucketName, testName, type, specName);
-    }
+	@Test
+	@AllowedFFDC // The tested deployment exceptions cause FFDC so we have to allow for this.
+	public void launchOpenTracing20Tck() throws Exception {
+		
+		TCKRunner.build(server, Type.MICROPROFILE, "Open Tracing")
+			.withDefaultSuiteFileName()
+			.runTCK();
+	}
```

Comment:
Converting this whole file to tabs is certainly better than leaving it as a mix, but I'm unsure if it was deliberate as the other files in the project appear to use spaces.